.travis.yml

```diff
@@ -8,8 +8,10 @@ env:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+  - RAILS_ENV=test
+  - PARALLEL_TEST_PROCESSORS=2
   matrix:
-  - TEST_SUITE=vmdb
+  - TEST_SUITE=vmdb PARALLEL='^spec/(controllers|helpers|initializers|javascripts|lib|mailers|models|presenters|requests|routing|services|task_helpers|tools|views)'
```

Comment:
I kind of like the idea of bundling by suite...one downside is that rspec-rails is expecting a certain path for things like models and controllers, so it known when to include its helper methods.  We'd have to tweak the settings to recognize the deeper paths.